robottelo/config/facade.py

```diff
@@ -185,14 +182,18 @@ def configure_nailgun(self):
         def patched_gpgkey_init(self, server_config=None, **kwargs):
             """Set a default value on the ``content`` field."""
             gpgkey_init(self, server_config, **kwargs)
+            from robottelo.config import robottelo_root_dir
```

Comment:
This code won't be living for very long.

Reply:
the entire facade :)